src/dotnet/APIView/APIViewWeb/HostedServices/ReviewBackgroundHostedService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using APIViewWeb.Respositories;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+
+namespace APIViewWeb.HostedServices
+{
+    public class ReviewBackgroundHostedService : IHostedService, IDisposable
+    {
+        private bool _isDisabled = false;
+        private ReviewManager _reviewManager;
+
+        public ReviewBackgroundHostedService(ReviewManager reviewManager, IConfiguration configuration)
```

Comment:
Does this mean for this setting to be updated you need to restart the app (not sure on the lifetime of ```ReviewBackgroundHostedService```.

Reply:
Yes. Background service only run once when system is restarted. If a parser is changed then restart is required to deploy those changes so background task needs to be triggered only once. Configuration is added to support disabling this run in case we need to.